ansible_collections/arista/avd/examples/dual-dc-l3ls/README.md

```diff
@@ -0,0 +1,609 @@
+17. `evpn_gateway` configures the EVPN DC GW features that will be inherited by the children of this group, in this case dc2-leaf2a and dc2-leaf2b. `evpn_l2` configures EVPN DC GW for EVPN type 2 routes (MAC-IP) while `evpn_l3` configures the GW for EVPN type 5 routes (IP-PREFIX).
+18. `remote_peers` defines the RS for EVPN DC GW that will be configured on the device. This is a unique definition per device, and using the hostname, AVD can get all the information from the device in order to generate the configuration: Router ID to peer and BGP AS.
+
+Since we are adding the EVPN DC GW functionality in DC2, we need to also add it in DC1. This is a snipped part of `ansible-avd-examples/dual-dc-l3ls/group_vars/DC1.yml` file where the changes need to occur:
+
+```yaml title="DC1.yml"
+    DC1_L3_LEAF2:
+      bgp_as: 65102
+      evpn_gateway:
+        evpn_l2:
+          enabled: true
+        evpn_l3:
+          enabled: true
+          inter_domain: true
+      nodes:
+        dc1-leaf2a:
+          id: 3
+          mgmt_ip: 172.16.1.103/24
+          uplink_switch_interfaces:
+            - Ethernet3
+            - Ethernet3
+          evpn_gateway:
+            remote_peers:
+              - hostname: dc2-leaf2a
+        dc1-leaf2b:
+          id: 4
+          mgmt_ip: 172.16.1.104/24
+          uplink_switch_interfaces:
+            - Ethernet4
+            - Ethernet4
+          evpn_gateway:
+            remote_peers:
+              - hostname: dc2-leaf2b
+```
+
+Finally, more of the same, but this time for the L2 leaf switches:
+
+```yaml title="DC2.yml"
+l2leaf:
+  defaults:
+    platform: vEOS-lab
+    uplink_interfaces: ['Ethernet1', 'Ethernet2']
+    spanning_tree_mode: mstp
+
+  node_groups:
+    DC1_L2_LEAF1:
+      uplink_switches: ['dc2-leaf1a', 'dc2-leaf1b']
+      nodes:
+        dc1-leaf1c:
+          id: 11
+          mgmt_ip: 172.16.1.251/24
+          uplink_switch_interfaces:
+            - Ethernet8
+            - Ethernet8
+
+    DC1_L2_LEAF2:
+      uplink_switches: ['dc2-leaf2a', 'dc2-leaf2b']
+      nodes:
+        dc1-leaf2c:
+          id: 12
+          mgmt_ip: 172.16.1.252/24
+          uplink_switch_interfaces:
+            - Ethernet8
+            - Ethernet8
+```
+
+As should be clear, an L2 leaf switch is much simpler than an L3 switch. Hence there are fewer settings to define.
+
+## Specifying network services (VRFs and VLANs) in the EVPN/VXLAN fabric
+
+This file is identical to the one provided in the previous example. VRFs and VLANs are configured on all devices since no `tags` or `filter` are being used.
+
+It is important to consider adding the new leaves to the Inventory as described in the [Inventory](#content-of-the-inventoryyml-file) section, so they will rely on this file to configure the network services.
+
+```yaml title="NETWORK_SERVICES.yml"
+--8<--
+examples/dual-dc-l3ls/group_vars/NETWORK_SERVICES.yml
```

Comment:
Is rendering weird or is this on purpose? Seems like some content is missing?

Reply:
@nielsjlarsen What does your view look like? It seems to render normally when I run mkdocs serve locally